zammad/values.yaml

```diff
@@ -399,26 +397,26 @@ initContainers: []
   #     - name: help-zammad
   #       mountPath: /opt/zammad
 
-# can be used to add additional containers / sidecars
-sidecars: []
-  # - name: s3-backup
-  #   image: some-aws-s3-backup:latest
-  #   env:
-  #     - name: AWS_DEFAULT_REGION
-  #       value: "eu-central-1"
-  #     - name: AWS_ACCESS_KEY_ID
-  #       value: "xxxxxxxxxxxx"
-  #     - name: AWS_SECRET_ACCESS_KEY
-  #       value: "xxxxxxxxxxxx"
-  #     - name: SYNC_DIR
-  #       value: "/opt/zammad"
-  #     - name: AWS_SYNC_BUCKET
-  #       value: "some-backup-bucket"
-  #     - name: AWS_SYNC_SCHEDULE
-  #       value: "0 * * * *"
-  #   volumeMounts:
-  #     - name: help-zammad
-  #       mountPath: /opt/zammad
+# # can be used to add additional containers / sidecars
+# sidecars: []
+#   # - name: s3-backup
+#   #   image: some-aws-s3-backup:latest
+#   #   env:
+#   #     - name: AWS_DEFAULT_REGION
+#   #       value: "eu-central-1"
+#   #     - name: AWS_ACCESS_KEY_ID
+#   #       value: "xxxxxxxxxxxx"
+#   #     - name: AWS_SECRET_ACCESS_KEY
+#   #       value: "xxxxxxxxxxxx"
+#   #     - name: SYNC_DIR
+#   #       value: "/opt/zammad"
+#   #     - name: AWS_SYNC_BUCKET
+#   #       value: "some-backup-bucket"
+#   #     - name: AWS_SYNC_SCHEDULE
+#   #       value: "0 * * * *"
+#   #   volumeMounts:
+#   #     - name: help-zammad
+#   #       mountPath: /opt/zammad
```

Comment:
What should we do with the sidecars?

Reply:
Not entirly sure. I guess it dependes on the uscase.
Maybe for now add it to the scheduler first as it has a fixed replica of 1, so the containers are not run multiple times. Lets see than what the user feedback is.

Needs to be mentioned in the upgrading instructions too.

Reply:
@monotek please see d03e08e75f2c06cce7a0f91c714359eaaaceabc9. I added a separate, optional deployment `zammad-sidecars` which has `replicas: 1` fixed and is only created if there are any sidecars present. Would you agree to this solution?

Reply:
Moved sidecar settings to the deployments.